### ISSUE TITLE
python3 detection in webserver should check both stderr and stdout on both platforms

### ISSUE THREAD

**Issue by jedirv** [SCAII/SCAII#77]

**Comment by khlam** [comment]
@jedirv Testing 0d4d0ba  on a Windows 10 and MacOS machine shows that python3 is now correctly detected. Can you verify that 0d4d0ba  doesn't error on your system?

**Comment by LinearZoetrope** [comment]
Verify that it works with conda envs, since that was a major issue.

**Comment by jedirv** [comment]
Was too "in the zone" to break out and try this on Friday - will do on Monday unless get a chance prior.  Almost have switching between replay files working on the "select" branch!

**Comment by jedirv** [comment]
I get this on my laptop (conda in play):

     Running `C:\Users\Jedi\.scaii\git\SCAII\target\debug\replay.exe webserver`
no python3 detected
ERROR - Python3 is required to run SCAII.

(C:\Program Files\Anaconda3) C:\Users\Jedi\.scaii\git\SCAII\core>


I peaked at the fix and it the implemented change was that run_command_read_stderr()  now reads from stdout rather than stderr, which will work on some cases, but not all (as in some cases the info is routed to stderr).  Instead, the fix should be to change webserver.rs such that 


#[cfg(windows)]
fn get_python_version(python_command: String) -> Option<String> {

and 

#[cfg(unix)]
fn get_python_version(python_command: String) -> Option<String> {

can be collapsed into a single (non-platform dependant) function, that tries both approaches:

first, call util::run_command(&python_command, args) and if that produces either a None or a Some(result_string) where result_string matches "Python 3",  then call util::run_command_read_stderr(&python_command, args) and see if there is a match there.

So, basically look everywhere on all platforms and I think we're good.

**Comment by khlam** [comment]
See  1241a86e53fdcb8b951473731cee8c1c27a7613c. 

After further research I understand that a bug in python 3.2 results in version information piped to stderr, which I assume everyone else is running, or some similar version. I've now properly tested running the webserver in python 3.5, 3.6, and 3.2 to confirm it behaves as expected. 

I'm not convinced run_command*_() is best used here, namely because we'll need to change it slightly just for this specific use, the built-in functions used in this commit will accomplish the same thing, and evoking system specific run_command*_() is not necessary.
